pages/linux/enum4linux.md

```diff
@@ -0,0 +1,24 @@
+# enum4linux
+
+> Enum4linux is a tool for enumerating information from Windows and Samba systems.
+> It attempts to offer similar functionality to enum.exe formerly available from www.bindview.com.
+
```

Comment:
Is there a homepage link we can include here?

Reply:
Yes there is here: https://github.com/portcullislabs/enum4linux